connexion/apis/flask_api.py

```diff
@@ -208,6 +208,9 @@ def _get_flask_response(cls, response, mimetype):
         if flask_utils.is_flask_response(response):
             return response
 
+        elif isinstance(response, tuple) and flask_utils.is_flask_response(response[0]):
```

Comment:
Here we are only checking if the first element in the tuple is already a flask response, along the lines of (<Response>, status_code, headers). If so, we send the full tuple to the underlying flask code which turns a returned tuple into a flask response: 

https://github.com/pallets/flask/blob/master/flask/app.py#L1698